src/mobile-ui-react/AcceptButton.scss

```diff
@@ -28,7 +28,7 @@
   width: 50px;
   border-radius: 25px;
   margin-left: -25px;
-  background-color: var(--buic-foreground-success);
+  background-color: var(--foreground-success);
```

Comment:
`--foreground-success` does not exist in mobile-ui-react, and while I assume it exists in your app, and I also assume that `--buic-foreground-success` no longer exists, this should be `--muic-foreground-success`, and a color definition for that should be added to MobileUi.scss. (If the color is different in dark mode, then a separate dark mode definition should also be provided. I'm guessing that it isn't different in dark mode.)

Reply:
Thanks for suggestions, I have updated PR.